src/components/App/App.tsx

```diff
@@ -29,6 +30,13 @@ const SellerPage = lazy(() => import('../SellerPage'));
 const MerchantsPage = lazy(() => import('../MerchantsPage'));
 const ErrorPage = lazy(() => import('../404Page'));
 
+const options = {
+  autoConfig: true, // set pixel's autoConfig
+  debug: true, // enable logs
+};
+const pixelId = '781939739005509';
```

Comment:
This isn't private right?

Reply:
ahh shoot thats a good call, this needs to be private

Reply:
Ok I changed it to be an env variable. Its probably best that we only set the env variable in prod, so we don't fire tracking events from local or staging.